src/models.py

```diff
@@ -98,8 +98,12 @@ def __init__(self, model, client):
                 if callable(attr):
                     self.should_index = attr
                 else:
-                    raise AlgoliaIndexError('{} should be a callable.'.format(
-                        self.should_index))
+                    should_index_attr_name = self.should_index
+                    if isinstance(attr, bool):
```

Comment:
Thank you for this review!
> at this point we don't know the type of attr, it could be a Django field or an unbound method  

I refactored in c833a40 to handle unbound methods as well, and added in 394970a the tests you suggested.

Does this look good to you?